src/html/shared/footer.html

```diff
@@ -0,0 +1,28 @@
+<footer class="c-footer">
+    <div class="c-footer--top">
+        <div class="o-wrapper">
+            <div class="o-layout">
+                <div class="o-layout__item">
+                    <br> {% include "./share.html" %}
+                    <br> {% include "./accordion.html" %}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="c-footer--bottom">
+        <div class="o-wrapper">
+            <ul class="o-list-inline">
+                <li class="o-list-bare__item o-list-inline__item c-footer--bottom_item">
+                    <a href="#" target="_blank">About</a>
+                </li>
+                <li class="o-list-bare__item o-list-inline__item c-footer--bottom_item">
+                    <a href="# " target="_blank">Privacy</a>
+                </li>
+                <li class="o-list-bare__item o-list-inline__item c-footer--bottom_item">
+                    <a href="# " target="_blank">Terms</a>
+                </li>
```

Comment:
I think you can also remove `o-list-bare__item` on individual `<li>` items

Reply:
@xavianaxw @xavierang correct-o mundo. I did take a look and `o-list-bare__item` does remove some margin from `li` but I think it's already removed from a reset elsewhere...

https://github.com/inuitcss/inuitcss/blob/develop/objects/_objects.list-bare.scss

Reply:
`generic.elements.scss` i believe